installer/templates/phx_live/live/home_live.ex

```diff
@@ -0,0 +1,31 @@
+defmodule <%= web_namespace %>.HomeLive do
+  use <%= web_namespace %>, :live_view
+
+  def render(assigns) do
+    <%= web_namespace %>.PageView.render("home.html", assigns)
+  end
+
+  def mount(_params, _session, socket) do
+    {:ok, socket}
+  end
+
+  def handle_params(_params, _uri, socket) do
+    {:noreply, socket}
+  end
+end
+
+defmodule <%= web_namespace %>.HomeLive.DepsCheck do
+  use <%= web_namespace %>, :live_component
+
+  def render(assigns) do
+    ~L"""
+    <h2>Results of <code>mix hex.outdated</code></h2>
+    <pre><%%= @deps %></pre>
+    """
+  end
+
+  def update(_assigns, socket) do
+    {deps, _} = System.cmd("mix", ["hex.outdated"])
```

Comment:
We probably want to do something like this:

```elixir
if not MyApp.Endpoint.config(:code_reloading) do
  raise "action disabled when not in development
end

{deps, _} = System.cmd("mix", ["hex.outdated"])
```

Just in case someone deploys the button to prod? 🤔

Reply:
I'm sure there will be many other ideas what component to ship with but anyway here's mine: live `IEx.Helpers.runtime_info()`, it can show interactivity (at least uptime changes every 1s) and doesn't require Mix so it's safe to ship to prod.

Reply:
agree. I also had concerns with folks deploying to prod :D , but it wasn't likely to stay. I like @wojtekmach idea! I will give it a try

Reply:
glad you like it! If you want to be fancy there could be a list of checkboxes:

```
[x] system [x] memory [x] limits [ ] allocators [ ] applications 
```

that is transformed into list of atoms passed to runtime_info/1 :D

Reply:
It could also be a radio button that shows one of them. One is always enabled. :)